src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -135,6 +135,10 @@ const DONATION_DONOR_TYPE_ENUM = Object.freeze({
     CONTACT1: 'Contact1'
 });
 
+const FORM_STATE_IMMUTABLE_FIELDS_BY_API_NAME = [
```

Comment:
This is minor, but the "by api name" here implies a map to me. Maybe FORM_STATE_IMMUTABLE_FIELDS_API_NAMES?

Reply:
Sounds great to me. Thanks for the feedback here. Made the change!